src/status_im2/contexts/chat/messages/view.cljs

```diff
@@ -84,8 +74,5 @@
 (defn chat
   []
   (reagent/create-class
-   {:component-did-mount    (fn []
-                              (rn/hw-back-remove-listener navigate-back-handler)
-                              (rn/hw-back-add-listener navigate-back-handler))
-    :component-will-unmount (fn [] (rn/hw-back-remove-listener navigate-back-handler))
+   {:component-will-unmount (fn [] (rf/dispatch [:chat/close]))
```

Comment:
i don't remember why but this won't work as well, there are lots of edge cases., the only way is to dispatch on swipe :(

Reply:
> the only way is to dispatch on swipe :(

Any suggestion on how to detect that?

Reply:
Means is there any event, or have to use gesture handler etc,?

Reply:
no probably its not possible, and i disabled swipe, but someone enabled it

Reply:
btw, if its working for now, can we keep this solution for release? Because once https://github.com/status-im/status-mobile/pull/15893 is merged, we will have our own navigation for this screen, so don't need to rely on these events.

Reply:
Please can you create a PR for disabling swipe for ios, or point me in the direction. I am not finding option in navigation docs.

Reply:
https://wix.github.io/react-native-navigation/api/options-root/#popgesture

Reply:
Thank you